identifiers/country-et/states.csv

```diff
@@ -0,0 +1,13 @@
+id,name
+ocd-division/country:et,Ethiopia
+ocd-division/country:et/regional_state:af,Afar

```

Comment:
Why `regional_state` and not `state`?

Reply:
@jpmckinney , using `state` definitely is logical in this instance, our dilemma was that almost all official references [like this one](http://www.mfa.gov.et/Home/TemplateI?PageName=Government&TYPE=About%20Ethiopia&SUBTYPE=About%20Ethiopia&Language=English)  refer to these admin regions as  `regional states`  cc @jloutsenhizer

Reply:
I can go either way - I'd like the opinion of a second reviewer.

Reply:
My preference would be for `state` rather than `regional_state` since it'd reduce the number of unique identifier types for first-level adminstrative areas.